extras/docker/Dockerfile.buildtest

```diff
@@ -14,6 +14,7 @@ RUN true \
  && dnf -y install \
            git autoconf automake gcc libtool make \
            glusterfs-api-devel libuuid-devel json-c-devel \
+           tcmu-runner targetcli \
```

Comment:
This is not needed for building at all.
Only needed for running gluster-block.
And we can't do runtime tests here because the kernel in travis ubuntu 14.04 is not sufficient, afaik.

Reply:
@obnoxxx probably, you didn't noticed the commit msg/subject-line.

This was an effort to verify if tcmu-runner and targetcli are available, nothing more than that.

Reply:
Yeah, I noticed the message. But the build does not require it right now, and the container is never run, so no point in adding it just now. :-)